test/acceptance/features/steps/postgres_db.py

```diff
@@ -24,6 +24,16 @@ class PostgresDB(object):
   image: docker.io/postgres
   imageName: postgres
   dbName: {name}
+  tags:
+          - "centos7-12.3"
+          - 123
+  userLabels:
+          archive: "false"
+          environment: "demo"   
```

Comment:
```suggestion
          environment: "demo"
```
Trailing white spaces. Lint error.